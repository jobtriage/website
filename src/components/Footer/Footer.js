import React from 'react';
import { css } from '@emotion/core';
import Typography from '../../UI/Typography/Typography';
import { useTheme } from 'emotion-theming';

const Footer = () => {
    const theme = useTheme();
    const root = css({
        div: {
            width: '100vw',
            height: '1.5px',
            backgroundColor: '#f3f3f3'
        },
        section: {
            margin: '30px auto',
            display: 'flex',
            justifyContent: 'space-between',
            '@media (max-width: 850px)': {
                display: 'block',
                margin: '30px auto',
                width: '95vw',
                textAlign: 'center',
                span: {
                    display: 'block',
                    margin: '20px auto'
                }
            }
        },
        a: {
            color: theme.palette.secondary.main
        }
    })
    return (
        <div css={root}>
            <div></div>
            <section>
                <Typography component='p'>
                    Released v0.1.0 under MIT License. Copyright &copy; 2020 Job Triage.
                </Typography>
                <span>Follow <a href='https://twitter.com/KoushikKM96' target='_blank'>@Twitter</a> to get updates!</span>
                <span>Need help? <a href='mailto:jobtriage@gmail.com' target="_blank">@Gmail</a></span>

            </section>

        </div>

    )
};

export default Footer;