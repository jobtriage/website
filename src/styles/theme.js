const theme = {
  palette: {
    primary: {
      main: '#794BC4',
      dark: '#482B8A',
      light: '#BA9DE9',
    },
    secondary: {
      main: '#B041BF',
      light: '#853094',
      dark: 'DD95E7',
    },
    tertiary: {
      main: '#515CC6',
      light: '1A299C',
      dark: 'A1ABEA',
    },
  },
};

export default theme;
