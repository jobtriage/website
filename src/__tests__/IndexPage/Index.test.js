import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'emotion-theming';
import serializer from 'jest-emotion';
import theme from '../../styles/theme';
import Footer from '../../components/Footer/Footer';


expect.addSnapshotSerializer(serializer);
test('should render without crash', () => {
  const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <Footer />
    </ThemeProvider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
