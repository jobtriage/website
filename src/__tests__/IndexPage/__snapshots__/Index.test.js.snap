// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crash 1`] = `
.emotion-1 div {
  margin-top: 5em;
  height: 2px;
  background-color: #f3f3f3;
}

.emotion-1 section {
  margin: 30px auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media (max-width:850px) {
  .emotion-1 section {
    display: block;
    margin: 30px auto;
  }

  .emotion-1 section span {
    display: block;
    margin: 20px auto;
  }
}

.emotion-1 a {
  color: #B041BF;
}

.emotion-0 {
  font-size: 16px;
  line-height: 24px;
  font-weight: 400;
  color: #3f3d3d;
}

.emotion-1 div {
  margin-top: 5em;
  height: 2px;
  background-color: #f3f3f3;
}

.emotion-1 section {
  margin: 30px auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media (max-width:850px) {
  .emotion-1 section {
    display: block;
    margin: 30px auto;
  }

  .emotion-1 section span {
    display: block;
    margin: 20px auto;
  }
}

.emotion-1 a {
  color: #B041BF;
}

.emotion-0 {
  font-size: 16px;
  line-height: 24px;
  font-weight: 400;
  color: #3f3d3d;
}

<div
  className="emotion-1"
>
  <div />
  <section>
    <p
      className="emotion-0"
    >
      Released v0.1.0 under MIT License. Copyright Â© 2020 Job Triage.
       
    </p>
    <span>
      Follow
      <a
        href="https://twitter.com/KoushikKM96"
        rel="noopener noreferrer"
        target="_blank"
      >
        @Twitter
      </a>
       
      to get updates!
    </span>
    <span>
      Need help?
      <a
        href="mailto:jobtriage@gmail.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        @Gmail
      </a>
    </span>
  </section>
</div>
`;
